app/assets/stylesheets/components/_index.scss

```diff
@@ -1,3 +1,11 @@
 // Import your components CSS files here.
 @import "alert";
 @import "navbar";
+@import "card";
+
+h1 {
+  font-size: 100px;
+}
+.no-chart-info-text {
```

Comment:
no-chart-info-text? Can we pick a better class name maybe?

Reply:
Agree about the h1, should be nested within .no-chart-info-text (assuming that's where it appears) so it does not interfer with another class

Reply:
The part "selectors" on this page can be helpful: http://htmlcheatsheet.com/css/